src/afl-fuzz-init.c

```diff
@@ -1056,6 +1056,13 @@ void perform_dry_run(afl_state_t *afl) {
               "skipping",
               fn, (int)(s8)afl->fsrv.crash_exitcode);
 
+        } else if (afl->crashing_seeds_as_new_crash) {
```

Comment:
you need to take care of a crash with uses_crash_exitcode as well (above)

Reply:
basically, just reverse the if and else parts

Reply:
moved it to inside of else statement